Library/Homebrew/locale.rb

```diff
@@ -68,6 +68,11 @@ def eql?(other)
   end
   alias == eql?
 
+  def detect(others)
```

Comment:
Use `*others` here and `locale.detect(*@language_blocks.keys)` above.

Reply:
That was my first try, but I got `Error: '["en"]' cannot be parsed to a Locale`.

Reply:
Ah right, the keys are actually arrays.